tests/bm_test.py

```diff
@@ -163,7 +161,11 @@ def eval_c2st(
 
 
 def train_and_eval_amortized_inference(
-    inference_class, task_name: str, extra_kwargs: dict, results_bag: ResultsBag
+    inference_class,
+    task_name: str,
+    extra_kwargs: dict,
+    results_bag: ResultsBag,
+    benchmark_num_simulations: int,
```

Comment:
I like making the num_simulations an explicit argument, but I would move this up next to `task_name` as opposed to putting it at the end of the args